test/models/mxnet/mxnet_densenet121_inference_batch1_float32.json

```diff
@@ -0,0 +1,9398 @@
+[{
```

Comment:
Sorry, I don't think I can provide too much feedback on this file. Let me know if you want me to look at something in particular.

Reply:
Thanks, Diego!